### PR TITLE
update week 4 content

### PR DESCRIPTION
**Pull request by tvpeter**
- Remove unused week-1 content
- Update Week 4 content
- Add exercises for Week 4

### CODE REVIEW

File: 01-2024-schedule/course-schedule/week-4.md [hunk 1]
```diff
@@ -7,3 +7,66 @@ description: Second-Layer Applications
...
+
+
+| Topic        | Goals               | Resources                                                         |
+| ------------ | --------------------|------------------------------------------------------------------ |
+| Lightning Network Basics     | Understand: <br/> Why Lightning is called Layer 2 <br/> Trust in Decentralised Network <br/> Security Assumptions <br/> The Lightning Network components |[Chapter 1: Introduction](https://lnbook.256k1.dev/#intro_what_is_the_lightning_network) <br/> [How Network Layers Fit Together](https://youtu.be/krux2v0jt4E?list=PLpLH33TRghT17_U3as2P3vHfAGL8pSOOY) |
+| Lightning Nodes and Wallets   | Understand:  Lightning Nodes <br/> Wallets |[Lightning Nodes](https://lnbook.256k1.dev/#_lightning_nodes) <br/> [Lightning Wallets](https://lnbook.256k1.dev/#_lightning_wallets)                                                   |
+| Lightning Network Architecture    | Understand the various components of the Lightning Network |[Chapter 6: Lightning Network Architecture](https://github.com/lnbook/lnbook/blob/develop/06_lightning_architecture.asciidoc#lightning-network-architecture)                                                                  |
```
Comment (Extheoisah) [inline 01-2024-schedule/course-schedule/week-4.md:58]: Same here. This should come before Lightning Nodes and Wallets on line 57

**Review by Extheoisah — CHANGES_REQUESTED** [review]
Looks Good to me. Just a small comment

File: 01-2024-schedule/course-schedule/week-4.md (hunk 1 above)
Comment (tvpeter) [reply 01-2024-schedule/course-schedule/week-4.md:57]: Done.

**Review by Extheoisah — APPROVED** [review]

**Comment by NonsoAmadi10** [comment]
A quick question I want to ask, are we not going to allow fellows build lightning apps of theirs using polar especially so that they can see the beauty in building rapid lightning apps and test on the fly @tvpeter ?

**Comment by tvpeter** [comment]
> A quick question I want to ask, are we not going to allow fellows build lightning apps of theirs using polar especially so that they can see the beauty in building rapid lightning apps and test on the fly @tvpeter ?

I think the last two exercises are doing that.

**Comment by Extheoisah** [comment]
> A quick question I want to ask, are we not going to allow fellows build lightning apps of theirs using polar especially so that they can see the beauty in building rapid lightning apps and test on the fly @tvpeter ?

@enigbe and I will also go over it again to add/subtract from it.

File: 01-2024-schedule/course-schedule/week-4.md [hunk 2]
```diff
@@ -7,3 +7,66 @@ description: Second-Layer Applications
...
+  * Fairness Protocol
+  * Hash Time-Locked Contracts (HTLCs)
+* Understand Channel Operation and Payment Forwarding
+  * Managing HTLCs
+  * Committing HTLCs to channel state
+  * Channel balance
+* Understand Path Finding and Payment Delivery
+  * Finding Candidate Paths
+  * Payment Delivery
+  * Multiparts Payment (MPP)
```
Comment (Jennycj) [inline 01-2024-schedule/course-schedule/week-4.md:45]: Can this be Multiparts/Multipath Payment(MPP)? They are used interchangeably and might be confusing when searching for external resources.

Comment (tvpeter) [reply 01-2024-schedule/course-schedule/week-4.md:45]: Thank you for the observation. However, in the `lnbook`, it is stated that `the term multipart payment is more accurate than multipath payment` Check Multipart Payment (MPP) in the [glossary](https://lnbook.256k1.dev/#glossary).   AMP (Atomic Multipath Payment) is used more often when referring to Multipath.

**Pull request merged**